Gemfile

```diff
@@ -3,8 +3,13 @@ source 'https://rubygems.org'
 ruby '2.3.3'
 
 gem 'capybara'
+gem 'capybara-screenshot'
 gem 'dotenv'
 gem 'gmail'
 gem 'json'
 gem 'poltergeist'
+gem 'pry'
```

Comment:
Did you mean to leave `pry` and `capybara-screenshot` here?

Reply:
No, good catch, I was using that to debug but we definitely don't need all of that in the main bundle group.